src/commands/go-build.yaml

```diff
@@ -3,6 +3,11 @@ parameters:
     type: "string"
   os:
     type: "string"
+  path:
```

Comment:
You mean main.go right?

Reply:
The description is a bit unclear to me

Reply:
No I mean `.` as the import path to the package being built, as in `go build -v .`

Reply:
Should we not then use build-path?

Reply:
How can I improve the description ?